### ISSUE TITLE
Video for complete deployment of DeveloperFolio

### ISSUE THREAD

**Issue by saadpasta** [saadpasta/developerFolio#190]
## Summary
I have received lots of messages that people are facing issues in deploying their portfolio to Github pages.

### What's new?
We would create a simple video on how to clone our project and change details and deploy it to Github pages.

### Motivation
It would help lots of newcomers to deploy their portfolio without any hassle.

**Comment by kartik918** [comment]
A video walkthrough seems too much (maybe a youtube link if we want to do it), maybe a page on the "demo" portfolio that would be a step by step instruction with images to help set it up for deployment. Open to discussion on this.

**Comment by naveen521kk** [comment]
The video if required should be a YouTube link and according to me is way too much for this project. Also, create a wiki page for Deploying step by step with pictures would help instead of doing it in Readme file or the demo portfolio. The demo portfolio is intended for people 👫 to check how it actually works.

**Comment by saadpasta** [comment]
@naveen521kk @kartik918 so should we go with the readme with pictures.

**Comment by naveen521kk** [comment]
According to me, Readme looks pretty filled up. Rather, we could use GitHub Wiki for it. With pictures.

**Comment by kartikcho** [comment]
Pinging @saadpasta @naveen521kk for status on this issue. Anyone up for technical writing in the Wiki?

**Comment by naveen521kk** [comment]
> Anyone up for technical writing in the Wiki?

Currently very busy for rest of year :(( Sorry.

**Comment by WarenGonzaga** [comment]
@kartikcho I'm a technical writer at Vultr and I guess you could use my help.... also If you guys need a video for it I can create one but it is needed to be uploaded to my YouTube channel is it good for you? @kartikcho if it is good assign this task to me.

**Comment by kartikcho** [comment]
Hey @WarenGonzaga that's awesome! 
I think a video would be unconventional and a bit overkill. A detailed step by step guide for setup + deployment, with some tips in customizing the portfolio for the user would be really great! Thanks for offering your help, assigning this to you.

**Comment by WarenGonzaga** [comment]
Alright thank you @kartikcho the documentation should be on Github wiki right?

**Comment by kartikcho** [comment]
> the documentation should be on Github wiki right?

Yes. I don't have experience using GitHub Wiki for writing docs before so let me know if there's an access issue or something.

**Comment by WarenGonzaga** [comment]
Cool @kartikcho I'll let you know.

**Comment by WarenGonzaga** [comment]
Hi guys... sorry I'm a bit busy but still working on this.

**Comment by MrAsimZahid** [comment]
https://mrasimzahid.medium.com/how-to-deploy-developer-folio-on-github-pages-29c282f56749

If this is good, and the right way then I can convert it into the wiki.
This works for me. So, I wrote it
Thanks.

**Comment by kartikcho** [comment]
Hey @MrAsimZahid thanks for writing a blog to help people out! I took a look and it seems good for the wiki (although we could improve it by simplifying the terminal commands section and further explaining it for beginners).

The wiki should contain steps on deploying to <username>.github.io and if possible, explaining how to deal with `.env` file secrets so users stop accidentally pushing them to their public repos.

Assigning this to you!

**Comment by MrAsimZahid** [comment]
@kartikcho How can I push the wiki to the repository. I have completed writing it. Need to publish it.

**Comment by kartikcho** [comment]
@MrAsimZahid [this guide](https://gist.github.com/omaraboumrad/35654da0a376c57a2e0ab4d92ad0c339) should have everything you need to know. 
Let me know if there are any issues or if you're having trouble with this guide, I can publish the page to the wiki on your behalf (with proper attribution of course) and add you to contributors.

**Comment by AleixMT** [comment]
Yeah... I have been trying to deploy this awesome project as a `github.io` user page for an hour and I do not understand what do I need to do. There are a lot of guides linked in the `README.md` but it is not clear exactly what you need to do to deploy the project to your own page. I actually followed the instructions but the page is not deploying. I do not think a video is necessary, instead, you should include all the steps in order to use the project as your own in the `README.md` instead of linking to many guides that only explain a fraction of the process. I will open an issue anyway to see if someone can help me but I am happy to see that the contributors are aware of the poor state of the documentation that covers the basic usage of this project. Thank you for your effort.

**Comment by MrAsimZahid** [comment]
@AleixMT I faced the same issue when I was working for my own site. I hope this will help.

https://mrasimzahid.medium.com/how-to-deploy-developer-folio-on-github-pages-29c282f56749
